sync.go

```diff
@@ -9,17 +9,14 @@ import (
 	"time"
 
 	"github.com/decred/dcrd/addrmgr"
-	"github.com/decred/dcrd/rpcclient"
-	"github.com/decred/dcrwallet/chain"
-	"github.com/decred/dcrwallet/errors"
-	"github.com/decred/dcrwallet/p2p"
-	"github.com/decred/dcrwallet/spv"
-	"github.com/decred/dcrwallet/wallet"
+	"github.com/decred/dcrwallet/errors/v2"
+	"github.com/decred/dcrwallet/p2p/v2"
+	w "github.com/decred/dcrwallet/wallet/v3"
+	"github.com/raedahgroup/dcrlibwallet/spv"
 )
 
 type syncData struct {
-	mu        sync.Mutex
-	rpcClient *chain.RPCClient
+	mu sync.RWMutex
```

Comment:
Please add a comment on the properties of this struct that this mutex lock is expected to protect. Below is an example of what I mean:
https://github.com/decred/dcrwallet/blob/3dcfaf5f8abc0f1d4956bececb51819b57a831ef/spv/sync.go#L43-L45